assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -401,3 +401,17 @@ che.infra.openshift.bootstrapper.binary_url=http://che-host:${CHE_PORT}/agent-bi
 che.infra.openshift.bootstrapper.timeout_min=10
 che.infra.openshift.bootstrapper.installer_timeout_sec=180
 che.infra.openshift.bootstrapper.server_check_period_sec=3
+
+# Defines whether use the Persistent Volume Claim for che workspace needs
+# e.g backup projects, logs etc or disable it.
+che.infra.openshift.pvc.enabled=true
+# Defines the name of Persistent Volume Claim for che workspace.
+che.infra.openshift.pvc.name=claim-che-workspace
+# Defines the size of Persistent Volume Claim of che workspace.
+# Format described here:
+# https://docs.openshift.com/container-platform/latest/dev_guide/compute_resources.html#dev-compute-resources
+che.infra.openshift.pvc.quantity=10Gi
```

Comment:
Mb it make sense to rename it to something like `che.infra.openshift.projects.pvc.name` while this pvc is for projects sources only